<launch>
  <!--param name="/use_sim_time" value="true"/-->
  <node name="rio_fg" type="aaucns_rio_fg_node" pkg="aaucns_rio" output="screen"/>
</launch>
